health/api/v1/health.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+
+import flask
+import requests
+
+
+health = flask.Blueprint("health", __name__)
+
+
+def get_blueprints():
+    return [
+        ["/health", health],
+    ]
+
+
+@health.route("/", defaults={"region": "all"})
+@health.route("/<region>")
```

Comment:
That's exactly how it will be, this blueprint will be later mapped at with `/health` as url_prefix
`app.register_blueprint(blueprint, url_prefix="/api/v1%s" % url_prefix) `